OpenProjects.html

```diff
@@ -2013,8 +2013,10 @@
 </p>
 
 <p>The projects in this page are open-ended. More specific projects are
-filed as unassigned enhancements in the <a href="http://bugs.llvm.org/">
-LLVM bug tracker</a>. See the <a href="http://bugs.llvm.org/buglist.cgi?keywords_type=allwords&amp;keywords=&amp;bug_status=NEW&amp;bug_status=ASSIGNED&amp;bug_status=REOPENED&amp;bug_severity=enhancement&amp;emailassigned_to1=1&amp;emailtype1=substring&amp;email1=unassigned">list of currently outstanding issues</a> if you wish to help improve LLVM.</p>
+filed as unassigned enhancements in the <a href="https://bugs.llvm.org/">
```

Comment:
```suggestion
filed as unassigned enhancements in the <a href="https://github.com/llvm/llvm-project/issues">
```

Reply:
Don't we want to redirect bugs.llvm.org to github.com/llvm/llvm-project/issues?

Reply:
probably but not sure why we should add a redirect when github provides a stable url?

Reply:
bugs.llvm.org will host Bugzilla. At least for quite some time from now.

Reply:
@sylvestre I like aliases since they only need updating in one place. However @asl's comment means there's no choice but to update the links. I'll update the PR.